core/designsystem/src/main/kotlin/team/ppac/designsystem/util/extension/NoRippleClickable.kt

```diff
@@ -21,4 +23,20 @@ fun Modifier.noRippleClickable(
         interactionSource = remember { MutableInteractionSource() },
         onClick = { multipleEventsCutter.processEvent(onClick) },
     )
+}
+
+@SuppressLint("ModifierFactoryUnreferencedReceiver")
+fun Modifier.rippleClickable(
+    rippleColor: Color,
+    enabled: Boolean = true,
+    debounceMillis: Long = 300L,
+    onClick: () -> Unit,
+): Modifier = composed {
+    val multipleEventsCutter = remember { MultipleEventsCutter(debounceMillis) }
+    clickable(
+        interactionSource = remember { MutableInteractionSource() },
+        enabled = enabled,
+        indication = rememberRipple(color = rippleColor),
+        onClick = { multipleEventsCutter.processEvent(onClick) },
+    )
```

Comment:
문득 든 생각인데 이거를 singleClickable로 따로 빼는거 어때?
```suggestion
fun Modifier.rippleClickable(
    rippleColor: Color,
    enabled: Boolean = true,
    debounceMillis: Long = 300L,
    onClick: () -> Unit,
): Modifier = composed {
    this then singleClickable(
        interactionSource = remember { MutableInteractionSource() },
        enabled = enabled,
        indication = rememberRipple(color = rippleColor),
        onClick = onClick,
    )
    
fun Modifier.singleClickable(
    ...
): Modifier = composed {
    clickable(...)
}
```

Reply:
좋은것같아~ 관용적으로 리플은 기본이니까 이름에 리플을 넣을 필요가 없을듯?

Reply:
좋습니당. Modifier에서 두가지 처리를 담당하는 걸 분리하자는 의미였슴다